STYLEGUIDE.md

```diff
@@ -88,7 +88,10 @@ Stick to the first few letters for abbreviations if they are sufficiently descri
 Further conventions are
 - `unit2unit`: convert between types or units, e.g. `mat2arr`: convert matrix to array or `s2ms`: convert seconds to milliseconds. Can also be used for simple adapter methods, along the lines of `filt2odefilt`.
 - `proj`: projection (if required: `projmat`, `projvec`, `projlinop`, ...)
-- `precond`: preconditioner
+- `precon`: preconditioner
+- `driftmat`: drift-matrix, `forcevec`: force-vector, `dispmat` dispersion-matrix, 
```

Comment:
this reflects the changes made in `statespace.SDE` and the respective subclasses. Now there is only one way of accessing those matrices/functions, under the present naming scheme.